### PR TITLE
More answered questions

### PR DESCRIPTION
**Pull request by JadeZYX**
1 change component name QandA to QACard and make it as individual entry for each QA box. 
2 complete functionality for display the first 4 QA and add more QA under QuestionsAndAnswers component. Handle this together as a whole part with the total number that should be displayed.

### CODE REVIEW

**Comment by JadeZYX** [comment]
[MORE ANSWERED QUESTIONS button](https://trello.com/c/6KZiTqht/116-more-answered-questions-button)

File: client/src/components/QuestionAnswerPart/QACard.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    }
+    return (b.helpfulness - a.helpfulness);
+  });
+  const [showModal, setShowModal] = useState(false);
+  const [questionId, setQuestionId] = useState(0);
+  const [questionText, setQuestionText] = useState('');
+  const [report, setReport] = useState(false);
+  const [displayedAns, setDisplayedAns] = useState(answerslist.slice(0, 2));
+  const [showCollapseBtn, setShowCollapeBtn] = useState(false);
+  const [showLoadMoreBtn, setShowLoadMoreBtn] = useState(true);
```
Comment (corneliusrenken) [inline client/src/components/QuestionAnswerPart/QACard.jsx:42]: Is this state still needed? I saw you have something similar in QuestionsAndAnswers.jsx

**Review by corneliusrenken — APPROVED** [review]
Looks good jade! 😃

**Comment by corneliusrenken** [comment]
[Load more Ans button](https://trello.com/c/szYsfaTK/115-load-more-ans-button)

**Pull request merged**